embedded/imager/configs/rpi-imager_defconfig

```diff
@@ -1,45 +1,3993 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Buildroot -gec35dcf Configuration
+#
+BR2_HAVE_DOT_CONFIG=y
+BR2_EXTERNAL_NAMES="IMAGER"
+BR2_EXTERNAL_IMAGER_PATH="/home/tdewey/src/rpi-imager/embedded/imager"
```

Comment:
Will this still build for users that don't have a `/home/tdewey/...` directory on their computer?

Reply:
I'll prune it, but I would expect it to be overridden at build-time. The correct way to build the Embedded Imager is to use the build.sh file, which should override these BR2_EXTERNAL directives with the buildroot invocation:

`make -C "${BUILDROOT}" BR2_EXTERNAL="${PWD}/imager" rpi-imager_defconfig`

Reply:
Ahhh, awesome; I didn't realise that :+1: